test/integration/consul-container/libs/service/assets/Dockerfile-consul-envoy

```diff
@@ -5,4 +5,18 @@ ARG ENVOY_VERSION
 FROM ${CONSUL_IMAGE} as consul
 
 FROM docker.mirror.hashicorp.services/envoyproxy/envoy:v${ENVOY_VERSION}
+
+# Install iptables and sudo, needed for tproxy.
+RUN apt update -y \
+	&& apt install -y iptables sudo curl jq dnsutils iproute2 \
```

Comment:
todo: some of these were for debugging. I need iptables and sudo. I used curl and dig (from dnsutils) but could potentially find alternatives. I don't need jq or iproute2.

Reply:
I removed jq and iproute2. I could have removed dnsutils by using nslookup, but nslookup doesn't have a way to return just the ip. Using dig is a bit nicer so I kept it for now. fwiw, I also tried `curl --dns-ipv4-addr` but the build of curl installed from the apt repo did not have the dns resolution functionality included.

 Ideally, we follow up on this eventually and configure it to use Consul DNS by default.